spyder/plugins/variableexplorer/widgets/namespacebrowser.py

```diff
@@ -209,15 +208,19 @@ def setup_toolbar(self):
             self, text=_("Search variable names and types"),
             icon=ima.icon('find'),
             toggled=self.show_finder)
-        config_shortcut(lambda: self.show_finder(set_visible=True),
-                        context='variable_explorer',
-                        name='search', parent=self)
+
+        CONF.config_shortcut(
+            lambda: self.show_finder(set_visible=True),
+            context='variable_explorer',
+            name='search',
+            parent=self)
 
         self.refresh_button = create_toolbutton(
             self, text=_("Refresh variables"),
             icon=ima.icon('refresh'),
             triggered=self.refresh_table)
-        config_shortcut(self.refresh_table,
+
+        CONF.config_shortcut(self.refresh_table,

```

Comment:
Please indent this code as you've done for all other calls to `CONF.config_shortcut` in this PR.